### ISSUE TITLE
Add integrations?

### ISSUE THREAD

**Issue by corbob** [yatli/coc-powershell#54]
**Summary of the new feature**

On the [vscode-powershell](https://github.com/powershell/vscode-powershell) repository there is integrations setup for Codacy and Dependabot. Would it be an idea to add those here as well? Dependabot at least would allow us to ensure we're keeping somewhat up to date on dependencies with hopefully less effort.

**Comment by TylerLeonhardt** [comment]
Yep. Agree. @yatli since you’re doing a few other coc extensions… maybe it’s worth making a `coc-extensions` github org and put this, fsharp, and omnisharp under it?

Then we can have better permissions.

**Comment by yatli** [comment]
yeah I'd very much love to consolidate things a bit.
I'll take some time to understand these modern toolings ;)

**Comment by TylerLeonhardt** [comment]
we have dependabot now :)
